### PR TITLE
nexus 7, oh how I hate you

### PR DESCRIPTION
**Pull request by ZeroChaos-**

### CODE REVIEW

File: prebuilt/common/bin/wia-ng.sh [hunk 1]
```diff
@@ -30,6 +30,10 @@ if [ -d /sys/class/net ]; then
       #check wlan cards
       if grep -q "DEVTYPE=wlan" "${i}/uevent"; then
         if [ -r "${i}/device/modalias" ]; then
+          # XXX TODO: better check here, is there a way to check for wifi direct mode?
+          if [ "${i}" = "/sys/class/net/p2p0" ]; then
+            continue
+          fi
```
Comment (tmossey) [inline prebuilt/common/bin/wia-ng.sh]:
```
shell@hammerhead:/ $ pm list features |grep -i direct                                                                                                                                                                                                                                      
feature:android.hardware.wifi.direct
```

Comment (tmossey) [reply prebuilt/common/bin/wia-ng.sh]: not sure how to check if it's in that mode

**Comment by dallaswinger** [comment]
"give it to me dallas"

**Comment by dallaswinger** [comment]
ack

**Pull request merged**